Source/WebCore/svg/SVGMaskElement.cpp

```diff
@@ -128,6 +136,14 @@ void SVGMaskElement::childrenChanged(const ChildChange& change)
     if (change.source == ChildChange::Source::Parser)
         return;
 
+#if ENABLE(LAYER_BASED_SVG_ENGINE)
+    if (document().settings().layerBasedSVGEngineEnabled()) {
+        if (auto* renderer = this->renderer())
+            renderer->repaintClientsOfReferencedSVGResources();
+        return;
+    }
+#endif
```

Comment:
Some helper method may be convenient if we start using this everywhere for resources.

Reply:
Yeah, we can think about a more central helper once the full pattern has emerged for more than just one or two resources.